src/encoded/tests/features/search.feature

```diff
@@ -60,41 +72,79 @@ Feature: Search
+        And I click the element with the css selector ".facet__expander--header[aria-label='assay_title']" 
+        And I wait for the content to load
         When I fill in "searchAssaytitle" with "dna"
         Then I should see at least 2 elements with the css selector "div.facet__term-list.searchAssaytitle > li"
 
     Scenario: Search for Target of Assay term
         When I press "Data"
+        And I wait for the content to load
         And I click the link to "/search/?type=Experiment&status=released&perturbed=false"
         And I wait for the content to load
+        And I click all elements with the css selector "[data-test='facetcontainer'] .facet__expander--header[aria-pressed='true']"
+        And I wait for the content to load
+        And I click the element with the css selector ".facet__expander--header[aria-label='target.label']" 
+        And I wait for the content to load
         When I fill in "searchTargetofassay" with "fkh-10"
         Then I should see at least 1 elements with the css selector "div.facet__term-list.searchTargetofassay > li"
 
     Scenario: Search for Organ term
         When I press "Data"
+        And I wait for the content to load
         And I click the link to "/search/?type=Experiment&status=released&perturbed=false"
         And I wait for the content to load
-        When I fill in "searchOrgan" with "zzz"
+        And I click all elements with the css selector "[data-test='facetcontainer'] .facet__expander--header[aria-pressed='true']"
+        And I wait for the content to load
+        And I click the element with the css selector ".facet__expander--header[aria-label='biosample_ontology.organ_slims']"
+        And I wait for the content to load
+        And I fill in "searchOrgan" with "zzz"
         Then I should see 0 elements with the css selector "div.facet__term-list.searchOrgan > li"
 
     Scenario: Search for different ChIP-seq
-        When I click the link to "?type=Experiment"
+        When I press "Data"
         And I wait for the content to load
-        When I click the link to "?type=Experiment&assay_title=TF+ChIP-seq"
+        And I click the link to "/search/?type=Experiment&status=released&perturbed=false"
+        And I wait for the content to load
+        And I click all elements with the css selector ".clear-filters-control a"
+        And I wait for the content to load
+        And I click all elements with the css selector "[data-test='facetcontainer'] .facet__expander--header[aria-pressed='true']"
+        And I wait for the content to load
+        And I click the element with the css selector ".facet__expander--header[aria-label='assay_title']"
+        And I wait for the content to load
+        And I click the link to "?type=Experiment&assay_title=TF+ChIP-seq"
         And I wait for the content to load
         Then I should see "Showing 9 of 9 results"
 
-        When I click the link to "?type=Experiment"
+        When I press "Data"
```

Comment:
I had a hard time gettin the old code to work. So, I re-wrote it to be more explicit.